node-tests/setup-middleware-test.js

```diff
@@ -42,6 +40,7 @@ QUnit.module('setupMiddleware', function (hooks) {
       urlPath: 'report-violations',
       reportDir: 'ember-a11y-report',
     });
+    let { default: getPort } = await import('get-port');
```

Comment:
would love to learn a better way of doing this kind of interop if it exists

Reply:
Using `import` is the way to do this type of thing. We could also just not upgrade to these breaking majors, and wait for esm to settle a bit more before converting. Either way works for me, and since this is only needed in tests I think this is fine.

Reply:
@scalvert totally agree that this is not a must at this point of time (and even maybe for another year or two, until old versions stop working, if it happens at all).

It's just something I noticed as was curios of the failure in dependabot PRs and went this rabbit hole.

Feel free to close as this actually may cause more churn that benefit.

Reply:
I'm totally fine with merging. Thanks!